src/external/IntelRDFPMathLib20U2/CMakeLists.txt

```diff
@@ -26,10 +26,7 @@ if(MSVC)
     target_compile_options(Bid PUBLIC /sdl-)
 endif()
 
-# Note: checking for -Wfoo since unknown -Wno-foo flags are ignored.
-check_cxx_compiler_flag(-Wunused-but-set-variable HAVE-Wunused-but-set-variable)
-if(HAVE-Wunused-but-set-variable)
-    target_compile_options(Bid PRIVATE -Wno-unused-but-set-variable)
-endif()
+# disable warnings in this project
+target_compile_options(Bid PUBLIC -w)
```

Comment:
Do we care that these changes are disabling warnings wholesale for these external libraries instead of explicitly calling out the ones to ignore?

Perhaps update the comment to include building the library with warnings enabled when upgrading to ensure there are no unhandled "gotchas" in the new version?